client/components/LearnerViewDirective.js

```diff
@@ -35,8 +35,10 @@ tie.directive('learnerView', [function() {
                   </div>
                   <div ng-class="{'tie-step-line': $index < (questionIds.length - 1)}"></div>
                 </div>
+                <div ng-class="{'tie-step-line': $index < (questionIds.length - 1)}"></div>
               </div>
             </div>
+          </div>
```

Comment:
Was this really removed? I am seeing a </div> on line 41, and I thought this was the </div> that was unnecessarily added?